### ISSUE TITLE
ReserveCacheError: Unable to reserve cache when re-running checks on PRs

### ISSUE THREAD

**Issue by misode** [ChenCMD/datapack-linter#9]
This warning showed up when clicking "re-run jobs" on a PR.
```
Warning: Failed to save the cache. The following errors may be resolved by reporting them in the datapack-linter repository.
Warning: ReserveCacheError: Unable to reserve cache with key datapack-linter-undefined-12-855219240, another job may be creating this cache.
```
See: https://github.com/Gamemode4Dev/GM4_Datapacks/pull/594/checks?check_run_id=2722492384

**Comment by ChenCMD** [comment]
This bug had already been fixed, but I stupidly forgot to release :(
We have released 1.3.1 now, so please use it. I think it will probably work fine
